kura/org.eclipse.kura.core.net/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,7 @@ Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.net;version="[2.0,3.0)",
  org.eclipse.kura.net.dhcp;version="[1.0,2.0)",
  org.eclipse.kura.net.firewall;version="[1.0,2.0)",
- org.eclipse.kura.net.modem;version="[2.0,2.1)",
+ org.eclipse.kura.net.modem;version="[2.0,3.0)",
```

Comment:
why we are enlarging the range here? core.net implements net.modem API, so I believe it should be [2.1,2.2), right?